library/ga.js

```diff
@@ -1,6 +1,16 @@
 import ReactGA from 'react-ga';
 
 const trackId = 'UA-126214096-1';
-ReactGA.initialize(trackId);
+const isProduction = process.env.NODE_ENV === 'production';
 
-ReactGA.pageview(window.location.hash);
+ReactGA.initialize(trackId, {
+    testMode: !isProduction,
```

Comment:
avoid negation, use the condition directly here `process.env.NODE_ENV !== 'production'`